webapp/utils/utils.jsx

```diff
@@ -686,24 +686,19 @@ export function applyFont(fontName) {
     }
 }
 
-export function changeCss(className, classValue, classRepeat) {
-    // we need invisible container to store additional css definitions
-    var cssMainContainer = $('#css-modifier-container');
-    if (cssMainContainer.length === 0) {
-        cssMainContainer = $('<div id="css-modifier-container"></div>');
-        cssMainContainer.hide();
-        cssMainContainer.appendTo($('body'));
+export function changeCss(className, classValue) {
+    let styleEl = document.querySelector('style[data-class="' + className + '"]');
```

Comment:
If you like, I'd like to look at the issue you mentioned again.. 
Adding the classValue to make selectors unique can cause weird behaviour when for example:
using ```url("http://www.google.com/logo.png")``` as a value, which breaks the quotation marks of the attribute selector. 
Additionally, when overwriting the same className with a different value, the <style> object will not get overwritten, it will get added as a new <style> element, which over time might cause a lot of <style> tags to be added in the <head> without cleaning them op properly. This can be tested by switching themes a lot with your fix..